signal_scanner_bot/twitter.py

```diff
@@ -67,15 +69,25 @@ def send_tweet(tweet: str, timestamp: datetime, api: tweepy.API) -> None:
         # If first tweet send without reply to tweet ID parameter, if part
         # of a thread send reply using ID of last tweet sent.
         if tweet_id is None:
-            status_obj = api.update_status(status=formatted)
-            tweet_id = status_obj.id
+
```

Comment:
In order to reduce duplicate code, could you wrap this entire `if...else` block in the `try...except`? Another thing to think about is if this error occurs in part of a multi-tweet chain - perhaps the `except` block should also have a `break` statement in it to prevent future messages in the thread from popping up?

Reply:
Oh yeah, good point. I will work on getting this wrapped around the for block.